libs/platform/src/lib/form/multi-input/multi-input.component.html

```diff
@@ -37,8 +37,7 @@
         [disabled]="disabled"
         [compact]="contentDensity === 'compact'"
         (keydown)="removeSelectedTokens($event)"
-        (addOnButtonClicked)="!mobile && addOnButtonClick()"
-        (click)="mobile && onPrimaryButtonClick(false)"
```

Comment:
@dmitry-stepanenko in mobile mode clicking on input should open full-screen window, currently I don't see this

Reply:
@dimamarksman  fixed